### PR TITLE
adding note about persistent connections for L7 log generation

### PR DESCRIPTION
**Pull request by penkeysuresh**
Product Version(s): CE v3.14 +
CC 3.15, 3.16

Issue:  [IA-561](https://tigera.atlassian.net/browse/IA-561), [RS-850](https://tigera.atlassian.net/browse/RS-850)

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

[IA-561]: https://tigera.atlassian.net/browse/IA-561?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RS-850]: https://tigera.atlassian.net/browse/RS-850?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: calico-enterprise/visibility/elastic/l7/configure.mdx [hunk 1]
```diff
@@ -115,6 +115,11 @@ In this step, you will configure L7 logs, select logs for collection, and test t
    kubectl annotate svc <service-name> -n <service-namespace> projectcalico.org/l7-logging-
    ```
 
+After annotating a service for L7 log collection, only newly established connections through that service will be proxied by Envoy, connections established before the service was annotated will not be proxied or interrupted, resulting in no log generation.
```
Comment (bmckercher123) [inline calico-enterprise/visibility/elastic/l7/configure.mdx]:
```suggestion
After annotating a service for L7 log collection, only newly-established connections through that service are proxied by Envoy. Connections established before the service is annotated are not proxied or interrupted, and no logs are generated.
```

File: calico-enterprise/visibility/elastic/l7/configure.mdx [hunk 2]
```diff
@@ -115,6 +115,11 @@ In this step, you will configure L7 logs, select logs for collection, and test t
...
+
+Conversely, when a service is deselected, any previous connections established through the annotated service will continue to be proxied by Envoy until they are terminated and will generate logs.
```
Comment (bmckercher123) [inline calico-enterprise/visibility/elastic/l7/configure.mdx]:
```suggestion
Conversely, when a service is deselected, any previous connections established through the annotated service continue to be proxied by Envoy until they are terminated, and logs are generated.
```

**Review by bmckercher123 — DISMISSED** [review]
This update shown is targeted for CE 3.17 only (root level). Is that the intention? The doc also needs to be updated for CC. Ticket indicates this goes back to 3.15. If this is 3.17 only, then we do not have 3.17 folders cut yet for CC until 3.16.1 goes GA.

**Review by tomastigera — DISMISSED** [review]

**Review by bmckercher123 — DISMISSED** [review]

**Review by bmckercher123 — DISMISSED** [review]
LGTM

Yes, copy to CC 3.15 and 3.16 (root level)

**Review by bmckercher123 — DISMISSED** [review]
LGTM

**Pull request merged**